packages/components/src/FormatFile/FormatFile.css

```diff
@@ -26,37 +27,100 @@
   height: 100%;
   box-sizing: border-box;
   padding: 0;
-  background-color: var(--file-card--header-background-color);
-  background-position: center;
+  border: var(--border-base);
+  border-radius: var(--radius-base);
+  background-color: var(--border-basebackground);
   background-repeat: no-repeat;
   background-size: cover;
   align-items: center;
   object-fit: cover;
   flex-shrink: 0;
 }
 
-.imageBlock::before {
+.imageBlockOverlay::before {
   content: "";
-  display: block;
-  position: absolute;
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
+  width: 100%;
+  height: 100%;
+  border: var(--border-base);
+  border-radius: var(--radius-base);
   background-color: var(--color-overlay);
   mix-blend-mode: multiply;
 }
 
 .icon {
-  margin: 0 auto;
+  margin: auto auto;
 }
 
-.progress {
+.fileContentWrapper {
+  display: flex;
+  width: inherit;
+  height: inherit;
+}
+
+.thumbnailFilename {
   display: flex;
   position: absolute;
-  top: 0;
+  bottom: var(--space-small);
+  left: var(--space-small);
+  font-size: var(--typography--fontSize-small);
+}
+
+.thumbnailFilenameSmall {
+  display: flex;
+  position: absolute;
+  bottom: var(--space-smaller);
+  left: var(--space-smaller);
+  font-size: calc(var(--base-unit) * 0.625);
+}
+
+.thumbnailNonImagedefault::before {
```

Comment:
Is it possible to resolve some of this duplicated styling like...

```
.thumbnailNonImage::before {
  content: "";
  display: block;
  position: absolute;
  right: 0;
  bottom: 0;
  z-index: 1;
  width: var(--space-base);
  height: var(--space-base);
  background: linear-gradient(
    270deg,
    rgb(244, 244, 244) 38.46%,
    rgba(244, 244, 244, 0) 100%
  );
}

.thumbnailNonImage.thumbnailLarge::before {
  width: var(--space-large);
  height: var(--space-large);
}
```

Off the top of my head I can't recall if you can stack selectors for pseudo elements, but if we can that would be ideal so we're not writing the same CSS twice with two lines difference.